odetoolbox/integrator.py

```diff
@@ -23,27 +23,24 @@
 import sympy
 import sympy.matrices
 import numpy as np
+from typing import Dict
 
 
 class Integrator():
     r"""
     Integrate a dynamical system by means of the propagators returned by ODE-toolbox (base class).
     """
-
-    def set_spike_times(self, spike_times):
+    def set_spike_times(self, spike_times: Dict[str, float]):  # spike_times is a dictionary
```

Comment:
```suggestion
    def set_spike_times(self, spike_times: Dict[str, float]):
```

Comment is maybe not needed; description of parameters should be in [function docstring](https://www.python.org/dev/peps/pep-0257/) below.